cmd/http/main.go

```diff
@@ -264,7 +264,6 @@ func (handler *KeyValueHandler) createKV(w http.ResponseWriter, r *http.Request)
 		if err != nil {
 			span.RecordError(err)
 			span.SetStatus(codes.Error, err.Error())
-			return "", fmt.Errorf("system backend failed to write data: %v", err)
```

Comment:
is there a reason to remove this line of returning on error?

Reply:
Added back. It might due to merge failures.